.github/workflows/data-transfer.yml

```diff
@@ -0,0 +1,479 @@
+          node-version: 14.x
+          
+      # CORDA NETWORK
+      - name: Generate github.properties
+        run: |
+          echo "Using ${GITHUB_ACTOR} user."
+          echo "username=${GITHUB_ACTOR}" >> github.properties
+          echo "password=${{ secrets.GITHUB_TOKEN }}" >> github.properties
+          echo "url=https://maven.pkg.github.com/${GITHUB_ACTOR}/weaver-dlt-interoperability" >> github.properties
+          cat github.properties
+        working-directory: tests/network-setups/corda
+        
+      - name: Start Corda Network
+        run: |
+          sed -i "/docker logs corda_partya_1 -f/"' s/^/#/' "scripts/start-nodes.sh"
+          make start &> corda-net.out &
+        working-directory: tests/network-setups/corda
+      
+      # FABRIC NETWORK
+      - name: Start Fabric Network
+        run: make start-interop
+        working-directory: tests/network-setups/fabric/dev
+        
+      - name: Corda Network logs
+        run: |
+          cat tests/network-setups/corda/corda-net.out
+          docker logs corda_partya_1
+        
+      # RELAY
+      - name: Edit Relay docker compose
+        run: make convert-compose-method2
+        working-directory: core/relay
+        
+      - name: Start Relay for network1
+        run: make start-server COMPOSE_ARG='--env-file docker/testnet-envs/.env.n1'
+        working-directory: core/relay
+      
+      - name: Start Relay for network2
+        run: make start-server COMPOSE_ARG='--env-file docker/testnet-envs/.env.n2'
+        working-directory: core/relay
+        
+      - name: Start Relay for Corda_Network
+        run: make start-server COMPOSE_ARG='--env-file docker/testnet-envs/.env.corda'
+        working-directory: core/relay
+        
+      # FABRIC DRIVER
+      - name: Setup Fabric Driver .env
+        run: |
+          sed -i "s#<PATH-TO-WEAVER>#${GITHUB_WORKSPACE}#g" docker-testnet-envs/.env.n1
+          sed -i "s#<PATH-TO-WEAVER>#${GITHUB_WORKSPACE}#g" docker-testnet-envs/.env.n2
+        working-directory: core/drivers/fabric-driver
+        
+      - name: Start Fabric Driver for network1
+        run: make deploy COMPOSE_ARG='--env-file docker-testnet-envs/.env.n1'
+        working-directory: core/drivers/fabric-driver
+      
+      - name: Start Fabric Driver for network2
+        run: make deploy COMPOSE_ARG='--env-file docker-testnet-envs/.env.n2'
+        working-directory: core/drivers/fabric-driver
+        
+      # CORDA DRIVER
+      - name: Start Corda Driver
+        run: make deploy COMPOSE_ARG='--env-file docker-testnet-envs/.env.corda'
+        working-directory: core/drivers/corda-driver
+        
+      # FABRIC CLI
+      - name: Setup Fabric CLI .npmrc
+        run: |
+          cp .npmrc.template .npmrc
+          sed -i "s/<personal-access-token>/${{ secrets.GITHUB_TOKEN }}/g" .npmrc
+          cat .npmrc
+        working-directory: samples/fabric/fabric-cli
+      - name: Build Fabric CLI
+        run: npm install
```

Comment:
Use `make build` here, which is what we mention in the docs too. You'll have to install `yarn` (using npm) in the prerequisites for this.

Reply:
Changing it to `make build`. I can add `yarn` but I'm thinking `yarn link` doesn't do anything wrt build or running the fabric-cli. `yarn link` is only when you want to import a node js package in another node js package.

Reply:
Changing it to `make build`. Doesn't `npm install` should install `yarn` also, otherwise I think docs needs to be updated as well?

Also I'm thinking `yarn link` doesn't do anything wrt build or running the fabric-cli. `yarn link` is only when you want to import a node js package in another node js package.
I know Readme suggests running `yarn link` and then running `fabric-cli`, but I don't think `yarn link` does anything.
Shall we remove it from `make build`?

Reply:
The prerequisites section explicitly asks the user to install Yarn. That instruction has been there from the beginning. Do we need anything else?

Reply:
Yeah, `yarn link` allows the `fabric-cli` to be available as a global executable. But you are right, we are only using it from the `bin` folder.

You can remove `yarn` if you want. Apart from the makefile and Readme, you'll also need to update all the "Setup...." pages in the docs and remove it from the prerequisites sections.